books/RayTracingInOneWeekend.html

```diff
@@ -1799,15 +1928,48 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [write-color-clamped]: <kbd>[color.h]</kbd> The multi-sample write_color() function]
 
-Update the camera class to incorporate multiple samples per pixel:
+So now we need to gather multiple samples for each pixel.
+We'll select samples from the area around the pixel center.
+Which begs the question: _what's the area surrounding the pixel_?
+
+It is very common to think of pixels as the square region around each pixel location, such that
+  these squares perfectly tile the pixel grid.
+However, there's no real reason why this must be the case.
+In fact, there's nothing that says that pixels have to sample all of the image area, or even that
+  they cannot have their regions overlap in some way.
+
+For example, consider the human eye: our "pixels" (photoreceptor cells) are randomly scattered with
+  varying density, respond to different wavelengths, cover differently sized areas, and frequently
+  overlap each other.
+(See figure 1 of [_Cell Populations of the Retina_](https://iovs.arvojournals.org/article.aspx?articleid=2188127)
+  for an example of the different photoreceptor types in the human retina. There are many more than
+  the single rod + three cone types that most people think of.)
+There's much more to say about this, but if you want to learn more about image sampling and
+  reconstruction, see the excellent technical memo
+  [_A Pixel is Not a Little Square, A Pixel is Not a Little Square, A Pixel is Not a Little Square_](https://www.researchgate.net/publication/244986797),
+  by Alvy Ray Smith, for a fantastic introduction.
+
+In spite of all this, we will proceed with the trivial square pixel model.
```

Comment:
From "Which begs the question: _what's the area surrounding the pixel_?" to here.

I'm not sure if the prose makes sense here.
It feels more like Rest of Your Life material than In One Weekend

It's just very deeply technical and is quite a large aside for something that we really don't have the ink to discuss in any amount of detail.